src/app/groups/group-details/group-details.component.ts

```diff
@@ -242,4 +252,42 @@ export class GroupDetailsComponent implements OnInit {
     });
   }
 
+  triggerUnsubscribeModal(){
+    $('#unsubscribe-modal').modal('show');
+    return false;
+  }
+
+  unsubscribeFromGroup(){
+    console.log("Group uid=",this.group.groupUid);
+    this.groupService.unsubscribeUser(this.group.groupUid).subscribe(resp => {
+      console.log("User removed from group",resp);
+      $('#unsubscribe-modal').modal('hide');
+      this.alertService.alert("group.unsubscribe-alert.text");
+      setTimeout(() => {
+        this.router.navigate(["/groups"]);
+      },2000);
```

Comment:
Rather navigate immediately. There's a parameter to the alert method that will keep the alert on screen even after navigation.